core-tests/shared/src/test/scala/zio/BracketTypeInferrenceSpec.scala

```diff
@@ -10,37 +10,37 @@ object BracketTypeInferrenceSpec {
   class E1 extends E
 
   def infersEType1: ZIO[R, E, B] = {
-    val acquire: ZIO[R, E, A]            = ???
-    val release: A => ZIO[R, Nothing, _] = ???
-    val use: A => ZIO[R, E1, B]          = ???
+    val acquire: ZIO[R, E, A]              = ???
+    val release: A => ZIO[R, Nothing, Any] = ???
```

Comment:
Doesn't that change the test itself?..

Reply:
As far as I remember, the point is that `acquire.bracket(release)(use)` doesn't require type annotations.